app/models/transcode_command.rb

```diff
@@ -12,7 +12,8 @@ class TranscodeCommand
     track_title: 'TITLE',
     track_number: 'TRACKNUMBER',
     album_title: 'ALBUM',
-    artist_name: 'ARTIST'
+    artist_name: 'ARTIST',
+    release_date: 'RELEASEDATE'
   }.freeze
```

Comment:
Not sure if this would be sent in as `DATE` rather than `RELEASEDATE` as I've named it here.
Will change once we're sure

Reply:
FLAC tag naming seems to be a bit of a free-for-all. As far as I can see the most widely recognised tag name for something like this is `DATE` (e.g. see [these docs](https://xiph.org/vorbis/doc/v-comment.html#fieldnames)) and the value should be *just a year*. The latter might make things slightly awkward with the current way the code works if the value has to be different for FLAC vs ID3 tags...

Reply:
Have changed to this to `DATE` as per the docs [here](https://xiph.org/vorbis/doc/v-comment.html#fieldnames)